src/app/pages/create-session/create-session.page.ts

```diff
@@ -173,8 +173,8 @@ export class CreateSessionPage implements OnInit {
         this.getImageUploadUrl(this.localImage);
       } else {
         const form = Object.assign({}, {...this.form1.myForm.getRawValue(), ...this.form1.myForm.value});
-        form.start_date = form.start_date.unix().toString();
-        form.end_date = form.end_date.unix().toString();
+        form.start_date = (Math.floor(form.start_date.unix() / 60) * 60).toString();
```

Comment:
Dividing with 60 and multiplying again with 60 gives u same answer. It won't change anythinf